web/test/pages/profile.test.tsx

```diff
@@ -2327,6 +2387,7 @@ describe("profile", () => {
           existingEmployees: undefined,
           municipality: undefined,
           homeBasedBusiness: undefined,
+          legalStructureId: randomPublicFilingLegalStructure(),
```

Comment:
Had to change it to have only randomPublicFilingLegalStructure() because some of the legal structures would make it fail.